lib/core/uploads/Upload.ts

```diff
@@ -1,8 +1,8 @@
-import { Contract, Shard } from "../shards/Shard";
+import { Contract, Shard } from '../shards/Shard';
 
 export interface Upload {
   id: string;
-  index: number;
+  index: string;
```

Comment:
`index` is a number that tells the order of the shard if multiple shards are used